cirq-ionq/cirq_ionq/ionq_client.py

```diff
@@ -170,6 +178,39 @@ def request():
 
         return self._make_request(request, {}).json()
 
+    def get_results(
+        self, job_id: str, sharpen: Optional[bool] = None, extra_query_params: Optional[dict] = None
+    ):
+        """Get job results from IonQ API.
+
+        Args:
+            job_id: The UUID of the job (returned when the job was created).
+            sharpen: Determines which error aggregation to use for debiased jobs.
```

Comment:
This explanation does not seem to match the argument since it is a bool and not a choice of enum or str.  I would explain this better.  Maybe something like "Whether to use sharpening in error aggregation.  Sharpening is ... or you can find more info here..."

(or maybe use the text in cirq-ionq/cirq_ionq/service.py)

Reply:
gotcha i updated the docstrings :)